build.sh

```diff
@@ -170,24 +175,27 @@ rm meilix-imclient_*_all.deb
 rm /sbin/initctl
 dpkg-divert --rename --remove /sbin/initctl
 
-# Unmount pseudo-filesystems within chroot
-umount -lf /proc
-umount -lf /sys
-umount -lf /dev/pts
-exit
+# Make sure we're exiting without an error code
+exit 0
```

Comment:
@Abhi2424shek Nope, I did this because I was trying to see the problem with `umount`, I forgot to remove this workaround.

https://travis-ci.org/yukiisbored/meilix/builds/186897548